### DISCUSSION TITLE
iNav LUA in Jumper T-Lite

### DISCUSSION THREAD

**Discussion by Kikerumbo** [EdgeTX/edgetx#2132] [LUA]
Greetings!

I'm trying to get the GPS coordinates to log into the radio, "in case of...".

I've found this LUA script ( https://github.com/moschotto/OpenTX_GPS_Telemetry ) and installed it in a Jumper T-Lite. I know it shows OpenTX, but the author specifies that will work too for EdgeTX.

I went to page 12/12 and on "screen1" I chose "script" and the name of the LUA ( GPSx9L ).

I can't get any telemetry screens to show and if I try to use the tool GPS Stats, it show a warning:

Syntax.error
GPS Stats x9L.lua:8
unexpected symbol near
<'

Any ideas or any other LUA I can use for this purpose?

Thanks!

**Comment by pfeerick** [comment]
How did you download it? It somewhat sounds like you downloaded the web page (since it mentions near the `<` character), not the actual lua script... Plus as you can see from the screenshot below that line 8 is actually blank as it's part of the comment block at the top of the file). 

Make sure you go into the actual file, and right click on the 'Raw' link to download it... 

![image](https://user-images.githubusercontent.com/5500713/179353677-ab0cfa22-bda1-483e-99d8-d3af3cab289e.png)
![image](https://user-images.githubusercontent.com/5500713/179353717-83f857f7-ed9c-494c-bce0-a8f3c97adb9e.png)

**Reply by Kikerumbo** [reply]
> How did you download it? It somewhat sounds like you downloaded the web page (since it mentions near the `<` character), not the actual lua script... Plus as you can see from the screenshot below that line 8 is actually blank as it's part of the comment block at the top of the file).
> 
> Make sure you go into the actual file, and right click on the 'Raw' link to download it...
> 
> ![image](https://user-images.githubusercontent.com/5500713/179353677-ab0cfa22-bda1-483e-99d8-d3af3cab289e.png) ![image](https://user-images.githubusercontent.com/5500713/179353717-83f857f7-ed9c-494c-bce0-a8f3c97adb9e.png)

Thanks sir.

You are totally right. I'm a noob in all things related to Github.

It totally works now.

Thanks!

**Reply by pfeerick** [reply]
Fantastic! Enjoy :) Yes, it can be a strange beast sometimes... 😁
